### ISSUE TITLE
Encrypt on demand panel / function

### ISSUE THREAD

**Issue by dwasyluk** [syscoin/syscoin#69]
To better leverage Syscoin's native encryption add a "cipher" panel to the escrow tab which allows for arbitrary data encryption using a given Syscoin address.

The pane should have two inputs- address and message. Address can be a v1, v2 address or an alias. When user presses the "Encrypt" button the contents of the message field should be encrypted using Syscoin encryption. There should also be a "Decrypt" button which reverses the process for the contents of the message field.

Decryption will only succeed if the decrypting user owns the private key related to the public key (address) used to encrypt the original message.

**Comment by sidhujag** [comment]
why not just use a syscoin message? this is exactly what its doing

**Comment by sidhujag** [comment]
For escrow its not possible because we are encrypting to the sellers address, when the escrow agent encrypts the same text, the cipher turns out to be different (can't compare) due to randomness in the encryption algorithm... so the cipher panel becomes useless.

**Comment by dwasyluk** [comment]
This was just based on our feature discussion - remember we discussed a on-demand encrypt/decrypt feature? Idea was I could encrypt something using your SYS public key (alias) and then send you the raw, encrypted content rather than having to send you an actual SYS message TX. I paste you the encrypted content, you put it in your "decrypt" panel and then can decipher it. You'd suggested the feature, I just created this item to track.
